### PR TITLE
docs: reorganize readme

### PR DESCRIPTION
**Pull request by ducanhpham0312**
Move scripts usage over manual `zbctl`

### CODE REVIEW

**Comment by ducanhpham0312** [comment]
Add comment from Slack

**Comment by ducanhpham0312** [comment]
Add comment from browser

File: README.md [hunk 1]
```diff
@@ -36,13 +36,6 @@ The following application should be installed:
 - Open Zeebe Simple Monitor at `localhost:8083`. No process should be visible yet, but the UI should be.
     - If not, restart `zeebe-simple-monitor` container in Docker Desktop until its log displays something like "Started ZeebeSimpleMonitorApp in x seconds".
 
-## Interaction with Zeebe broker using zbctl
-`zbctl` is a command-line tool to interact with Zeebe. The following examples illustrate how to use it.
-- Deploy a process: `zbctl --insecure deploy /path/to/bpmn/<process-name.bpmn>`
-- Create an instance for a process: `zbctl --insecure create instance <process-id> --variables <necessary variables>`. The process ID can be checked by either on the command line after deploying the process, or when opening the process' file in Camunda Modeler. Some variables are pre-defined under `./variables`.
```
Comment (ducanhpham0312) [inline README.md:42]: :+1:

**Review by marlaaks — COMMENTED** [review]
Test

**Comment by ducanhpham0312** [comment]
Comment from browser #2

**Review by ducanhpham0312 — COMMENTED** [review]
Test

**Review by marlaaks — COMMENTED** [review]
Another test

**Pull request closed without merging**